packages/eslint-config-expo/utils/react.js

```diff
@@ -0,0 +1,18 @@
+module.exports = {
+  parserOptions: { ecmaFeatures: { jsx: true } },
+  extends: ['plugin:react/recommended', 'plugin:react-hooks/recommended'],
```

Comment:
I'd like to remove `plugin:react/recommended` and list out a more minimal set of rules explicitly as a follow-up.

Reply:
Added a [task](https://linear.app/expo/issue/ENG-11850/%5Beslint-config-expo%5D-remove-pluginreactrecommended) so we could decide which rules to keep 👍